python/cudf_polars/tests/containers/test_dataframe.py

```diff
@@ -160,3 +162,49 @@ def test_empty_name_roundtrips_overlap():
 def test_empty_name_roundtrips_no_overlap():
     df = pl.LazyFrame({"": [1, 2, 3], "b": [4, 5, 6]})
     assert_gpu_result_equal(df)
+
+
+@pytest.mark.parametrize(
+    "arrow_tbl",
+    [
+        pa.table([]),
+        pa.table({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]}),
+        pa.table({"a": [1, 2, 3]}),
+        pa.table({"a": [1], "b": [2], "c": [3]}),
+        pa.table({"a": ["a", "bb", "ccc"]}),
+        pa.table({"a": [1, 2, None], "b": [None, 3, 4]}),
+    ],
+)
+def test_serialize(arrow_tbl):
+    plc_tbl = plc.interop.from_arrow(arrow_tbl)
+    df = DataFrame.from_table(plc_tbl, names=arrow_tbl.column_names)
+
+    header, frames = df.serialize()
+    res = DataFrame.deserialize(header, frames)
+
+    pl.testing.asserts.assert_frame_equal(df.to_polars(), res.to_polars())
+
+
+@pytest.mark.parametrize(
+    "arrow_tbl",
+    [
+        pa.table([]),
+        pa.table({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]}),
+        pa.table({"a": [1, 2, 3]}),
```

Comment:
Suggestion: let's move these tests that rely on the experimental submodule into `tests/experimental/...`

Reply:
done